src/ToDo/Business/ToDoTask.cs

```diff
@@ -21,11 +21,11 @@ internal ToDoTask(string listId, TaskData data)
 	{
 		ListId = listId;
 
-		Id = data.Id ?? throw new ArgumentNullException("data.Id", "Task must have a valid ID."); ;
+		Id = data.Id ?? throw new ArgumentNullException("data.Id", "Task must have a valid ID.");
 		Importance = data.Importance;
 		IsReminderOn = data.IsReminderOn;
 		Status = data.Status;
-		Title = data.Title;
+		Title = data.Title ?? data.DisplayName;
```

Comment:
Title no longer exists on the Task object in the beta api. Let's remove in favour of DisplayName

Reply:
Please check this: https://docs.microsoft.com/en-us/graph/api/todotasklist-post-tasks?view=graph-rest-beta&tabs=http
I see the response and the objet has a Title.

are you have another documentation where I can check when you request a task and get displayName?

Reply:
Yep, looks like the apis are a bit inconsistent. Let's leave this how it is for the moment